docs/docs/general/07-Troubleshooting/deprecated-endpoints.md

```diff
@@ -0,0 +1,14 @@
+# Deprecated Endpoints
+
+If you are using the askui library you may see this warning:
+
+`This endpoint will be deprecated. Please upgrade your askui version.`
```

Comment:
which warnning you want to use the one already in the inference or this?

Reply:
I will use the one which is already available